GiEnJul/Controllers/GiverController.cs

```diff
@@ -47,15 +48,22 @@ public async Task<ActionResult<PostGiverResultDto>> PostAsync([FromBody] PostGiv
                 return Forbid();
             }
 
+            var (giver_limit, active_event) = await _eventRepository.GetGiverLimitAndEventNameForLocationAsync(giverDto.Location);
+
             var giver = _mapper.Map<Giver>(giverDto);
             giver.EventName = await _eventRepository.GetActiveEventForLocationAsync(giverDto.Location);
 
             var insertedAsDto = _mapper.Map<PostGiverResultDto>(await _giverRepository.InsertOrReplaceAsync(giver));
 
+            var givers_in_event = await _giverRepository.GetGiversByLocationAsync(active_event, giverDto.Location);
```

Comment:
As this fetches ALL givers within a location, you also get givers which are already matched in this count. I think this will result in a counting error when showing a givers spot in the waiting list.

And as said in the meeting, all givers are in the "venteliste". Considering this i don't belive it's a good idea to give givers a waiting-list number, as it might cause confusion.